packages/client/wallets/aa/src/blockchain/wallets/EVMAAWallet.ts

```diff
@@ -212,24 +212,14 @@ export class EVMAAWallet extends LoggerWrapper {
         });
     }
 
-    public getSigner(type: "viem" = "viem"): {
```

Comment:
In AA land, "signer" usually applies to the underlying signing mechanism for an account. We even use it this way within `WalletConfig`. Check out [these docs](https://docs.pimlico.io/permissionless/how-to/accounts/use-kernel-account#create-the-signer) for `permissionless.js` the most popular library for AA.

`getSigner` conflicts with this, returning clients instead, likely confusing devs.

Reply:
Additionally, given the key role that ZD SDK plays within this repo, our SDK is built on top of `viem`. Rather than surfacing other ways to interact with our SDK through different libs (`ethers.js`, `web3.js`, etc) here, let's double down on `viem`, and ship separate adapters if **really** needed.

Reply:
Consider that `permissionless.js` the leading AA lib, and what the ZD SDK is built on, is built on `viem`

Reply:
This makes a lot of sense–"signer" is an overloaded term we're using because we also have the "signer" that you use for `getOrCreateWallet`
and it was coming from the ethers analogy, less relevant now